parameters added for Security Lake (#12724)
newrelic/docs-website: src/content/docs/logs/forward-logs/aws-lambda-sending-security-logs-s3.mdx

   <img title="AWS Lambda - Select region" alt="AWS Lambda - Select region" src={serverlessAWSLambdaSelectRegion}/>
3. Search for `newrelic` and check **Show apps that create custom IAM roles or resource policies** to find `newrelic-securitylake-s3-processor-LogForwarder`.
4. Click the `newrelic-securitylake-s3-processor-LogForwarder` details and click **Deploy**.

> 5. Follow the app guidance to set up necessary parameters:
> * SecurityLakeSQSArn,
> * SecurityLakeSubscriberRoleArn,
> * ExternalID,
> * NRLicenseKey.

Reviewer: ```suggestion
5. Follow the app guidance to set up necessary parameters:
* SecurityLakeSQSArn,
* `SecurityLakeSubscriberRoleArn`
* `ExternalID`
* `NRLicenseKey`
```

> First two you will get from your Security lake provider.

Reviewer: ```suggestion
You'll get `SecurityLakeSQSArn` and `SecurityLakeSubscriberRoleArn` from your Security lake provider.
```
